algorithms/ar-msrt/javascript/merge_sort.js

```diff
@@ -0,0 +1,36 @@
+function mergeSort(array) {
+    const merge = function (array1, array2) {
```

Comment:
Codacy found an issue: [Function has a complexity of 10. Maximum allowed is 5.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6334421)